src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
```

Comment:
Recommend creating a separate function like this:

```
function checkOnError(errorMessage: string, ...sides: number[]): void {
  if (sides.some(side => side <= 0)) {
    throw new Error(errorMessage);
  }
}
```

because this check is used multiple times